### PR TITLE
fix typo and reformat of cuda override docs

### LINKED ISSUE #1623: fix typos and corrections in previous PR
note to self to fix typos and formatting in previous PRs: #1601

### PR DESCRIPTION
**Pull request by ngam**
This is a follow up to PR #1601
Fixes #1623

This PR mainly fixes a typo: ~CONDA_CUDA_OVERRIDE~ should have been CONDA_OVERRIDE_CUDA

PR Checklist:

- [x] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] put any other relevant information below

### CODE REVIEW

File: src/user/tipsandtricks.rst [hunk 1]
```diff
@@ -137,18 +137,18 @@ To verify that the correct platform is being used, run the following commands af
...
      # OR
-     CONDA_CUDA_OVERRIDE="11.2" mamba install "tensorflow==2.7.0=cuda112*" -c conda-forge
+     CONDA_OVERRIDE_CUDA="11.2" mamba install "tensorflow==2.7.0=cuda112*" -c conda-forge
 
 .. note::
    
-   Note that you should select the cudatoolkit version most appropraite for your GPU; currently, we have "10.2", "11.0", "11.1", and "11.2" builds available, where the "11.2" builds are compatible with all cudatoolkits>=11.2. (At the time of writing, there seems to be a bug in how the cuda builds are resolved by mamba, defaulting to cudatoolkit==10.2; thus, it is prudent to be as explicit as possible like above or by adding ``cudatoolkit>=11.2`` or similar to the line above.) 
+  you should select the cudatoolkit version most appropraite for your GPU; currently, we have "10.2", "11.0", "11.1", and "11.2" builds available, where the "11.2" builds are compatible with all cudatoolkits>=11.2. (At the time of writing, there seems to be a bug in how the cuda builds are resolved by mamba, defaulting to cudatoolkit==10.2; thus, it is prudent to be as explicit as possible like above or by adding ``cudatoolkit>=11.2`` or similar to the line above.)
```
Comment (jaimergp) [inline src/user/tipsandtricks.rst:152]:
```suggestion
   You should select the cudatoolkit version most appropriate for your GPU; currently, we have "10.2", "11.0", "11.1", and "11.2" builds available, where the "11.2" builds are compatible with all cudatoolkits>=11.2. At the time of writing (Mar 2022), there seems to be a bug in how the CUDA builds are resolved by ``mamba``, defaulting to ``cudatoolkit==10.2``; thus, it is prudent to be as explicit as possible like above or by adding ``cudatoolkit>=11.2`` or similar to the line above. 
```

**Comment by ngam** [comment]
@jaimergp @conda-forge/core could you have another look at this relatively minor (but critical) correction? Thanks!

**Comment by ngam** [comment]
Imo this is a critical typo and should be fixed asap, so paging again @conda-forge/core

File: src/user/tipsandtricks.rst [hunk 2]
```diff
@@ -137,22 +137,21 @@ To verify that the correct platform is being used, run the following commands af
...
+   You should select the cudatoolkit version most appropriate for your GPU; currently, we have "10.2", "11.0", "11.1", and "11.2" builds available, where the "11.2" builds are compatible with all cudatoolkits>=11.2. At the time of writing (Mar 2022), there seems to be a bug in how the CUDA builds are resolved by ``mamba``, defaulting to ``cudatoolkit==10.2``; thus, it is prudent to be as explicit as possible like above or by adding ``cudatoolkit>=11.2`` or similar to the line above. 
 
+For context, installing the TensorFlow 2.7.0 CUDA-enabled variant, ``tensorflow==2.7.0=cuda*``, results in approximately 2 GB of packages to download while the CPU variant, ``tensorflow=2.7.0=cpu*``, results in approximately 200 MB to download. That is a significant bandwidth and storage wasted if one only needs the ``-cpu`` variant!
```
Comment (hmaarrfk) [inline src/user/tipsandtricks.rst:154]:
```suggestion
For context, installing the TensorFlow 2.7.0 CUDA-enabled variant, ``tensorflow==2.7.0=cuda*``, results in approximately 2 GB of packages to download while the CPU variant, ``tensorflow=2.7.0=cpu*``, results in approximately 200 MB to download. That is a significant bandwidth and storage wasted if one only needs the CPU only variant!
```

**Comment by hmaarrfk** [comment]
Thank you

**Pull request merged**